proto/ord_schema.proto

```diff
@@ -45,9 +61,15 @@ message ReactionInput {
     REACTION_ROLE_CATALYST = 4;
   }
   ReactionRole reaction_role = 2;
+  // Whether this species should be considered a limiting reactant.
   bool is_limiting = 3;
+  /**
+   * Used to define order of addition. ReactionInputs with the same 
+   * addition_stage were added simultaneously. One ReactionInput with a 
+   *  lower addition_stage than another was added earlier in the procedure.
+   */
   int32 addition_stage = 4;
-  float addition_time = 5;
+  float addition_time = 5;  // Reaction time (in hours)
```

Comment:
Should this have an explicit unit, `addition_time_in_h`?

Or, use a Time message here?

Reply:
Good catch